From 9b192b3366a48bec35cad374642e4a1f3398927e Mon Sep 17 00:00:00 2001
From: "Gary V. Vaughan" <gary@gnu.org>
Date: Mon, 3 Nov 2014 07:25:03 +0100
Subject: [PATCH 2/4] tests: avoid spurious test failure with libtool 2.4.3

Based on a report by Bruce Dubbs.

* tests/foreign.at (Libtool): Be tolerant of 'quote' replacing the
older `quote'.

Signed-off-by: Eric Blake <eblake@redhat.com>
---
 tests/foreign.at | 2 +-
 1 file changed, 1 insertion(+), 1 deletion(-)

diff --git a/tests/foreign.at b/tests/foreign.at
index 78230073..db85da31 100644
--- a/tests/foreign.at
+++ b/tests/foreign.at
@@ -57,7 +57,7 @@ AT_CHECK([./config.guess || exit 77], [], [ignore], [ignore])
 # Make sure at-path contains something valid, and let the test suite
 # display it when verbose.  And fail, skipping would too easily hide
 # problems.
-AT_CHECK([sed -n ["s,^.*\`\\(/[^']*\\)'.*,\\1,p"] stdout], [0], [stdout])
+AT_CHECK([sed -n ["s,^[^']*[\`']\\(/[^']*\\)'.*,\\1,p"] stdout], [0], [stdout])
 AT_CHECK([test -f "`sed -n 1p stdout`"])
 
 # Older libtoolize installed everything but install-sh...
-- 
2.13.0

